src/layout/Footer.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import { Center } from '@chakra-ui/react'
+
+const Footer = () => {
+  return <Center as='footer'>&copy; Lada496</Center>
+}
```

Comment:
I'd put this component under `components/` as it doesn't serve as a "layout" since it's just a dumb component that renders footer